examples/advanced-filter-convert-publish/res/docker/configuration.toml

```diff
@@ -23,6 +23,12 @@ Type = 'consul'
   Host = 'edgex-core-data'
   Port = 48080
 
+  [Clients.Logging]
+  Name = "edgex-support-logging"
```

Comment:
Why does the docker configuration.toml have this extra `Name` property?

Reply:
Note sure. Copied that from core-data's docker configuration.toml.

Reply:
No, must have been one of the other services that I copied it from.

Reply:
Ahh, it is that way in device-virtual. I am good with removing if not needed.

Reply:
Name doesn't exist on ClientInfo.

```
type ClientInfo struct {
	// Host is the hostname or IP address of a service.
	Host string
	// Port defines the port on which to access a given service
	Port int
	// Protocol indicates the protocol to use when accessing a given service
	Protocol string
}
```
I am going to remove it from the toml files.